### ISSUE TITLE
Add ability to override aria-label in carousel

### ISSUE THREAD

**Issue by erwinmombay** [ampproject/amphtml#442]
### First Timers Only
We know figuring out the process for contributing to an open source project can be intimidating, so we created this issue as a way for you to learn the ropes.  (If you feel comfortable contributing to open source projects, please leave this issue for someone else.)

This particular starter issue is designed to familiarize you with the AMP Code Contribution process by adding a simple feature to one of AMP components.

### Background
`<amp-carousel>` component has two buttons for navigating to next/previous slides. It uses `aria-label` to provide alternative text for these buttons as needed for accessibility. However the values for these labels are hard coded and not configurable.

![screenshot from 2017-04-28 13 28 16](https://cloud.githubusercontent.com/assets/2099009/25546355/976ddcba-2c17-11e7-9920-12da6efd23ef.png)

### Motivation
We like to allow page authors to optionally change the value of these labels for localization or customization purposes.

### What to do
 `amp-carousel` should accept two new attributes `data-next-button-aria-label` and  `data-previous-button-aria-label`. 

If any of these attributes are provided by the developer,  `amp-carousel` should use their values instead 
of the default strings when building the next/previous buttons in the `base-carousel.js` file.

### Step by step
- [ ] Claim this issue by adding a comment below.  Please only claim this bug if you plan on starting work in the next day or so.
- [ ] If you aren't too familiar with Git/GitHub, see the [Getting Started End-to-End Guide](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md) for [an intro to Git & GitHub,](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#intro-to-git-and-github) and [how to get a copy of the code](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#get-a-copy-of-the-amphtml-code).  You can also refer to the [Quick Start Guide](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-quick.md) for the necessary setup steps with less explanation than the End-to-End guide.
- [ ] Follow the instructions for [building AMP](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#building-amp-and-starting-a-local-server).
- [ ] [Create a Git branch](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#create-a-git-branch) for making your changes.
- [ ] [Commit your changes](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#edit-files-and-commit-them) frequently.
- [ ] [Push your changes to GitHub](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#push-your-changes-to-your-github-fork).
- [ ] [Create a Pull Request](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#send-a-pull-request-ie-request-a-code-review).  Mention `closes Issue #442` in the description.
- [ ] Assign @aghassemi as the reviewer
- [ ] [Respond to your reviewer's comments](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#respond-to-pull-request-comments) (if any).

Once approved, your changes will be merged.  **⚡⚡⚡Congrats on making your first contribution to the AMP Project!⚡⚡⚡**  You'll be able to see it [live across the web soon](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md)!

Thanks, and we hope to see more contributions from you soon.

### Questions?

If you have questions ask in this issue or on your Pull Request (mentioning @aghassemi) or see the [How to get help](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#how-to-get-help) section of the Getting Started guide.

**Comment by adelinamart** [comment]
Is this still needed?

**Comment by erwinmombay** [comment]
@adelinamart yes, but should be part of i18n work. not sure who owns that

**Comment by aghassemi** [comment]
Updated the feature to be a GFI.

**Comment by alannawalton** [comment]
I would like to claim this.
